examples/matRad_example8_photonsVMAT.m

```diff
@@ -0,0 +1,124 @@
+%% Example Photon Treatment Plan with VMAT direct aperture optimization
+%
+% %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+%
+% Copyright 2017 the matRad development team. 
+% 
+% This file is part of the matRad project. It is subject to the license 
+% terms in the LICENSE file found in the top-level directory of this 
+% distribution and at https://github.com/e0404/matRad/LICENSES.txt. No part 
+% of the matRad project, including this file, may be copied, modified, 
+% propagated, or distributed except according to the terms contained in the 
+% LICENSE file.
+%
+% %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+
+%%
+% In this example we will show 
+% (i) how to load patient data into matRad
+% (ii) how to input necessary parameters in the pln structure
+% (iii) how to setup a photon dose calculation
+% (iv) how to inversely optimize fluence directly from command window in MatLab.
+% (v) how to apply a sequencing algorithm
+% (vi) how to run a VMAT direct aperture optimization
+% (vii) how to visually and quantitatively evaluate the result
+
+%% Patient Data Import
+% Let's begin with a clear Matlab environment and import the head &
+% neck patient into your workspace.
+matRad_rc
+
+load('HEAD_AND_NECK.mat');
+
+%% Treatment Plan
+% The next step is to define your treatment plan labeled as 'pln'. This 
+% structure requires input from the treatment planner and defines 
+% the most important cornerstones of your treatment plan.
+
+pln.radiationMode   = 'photons';   % either photons / protons / carbon
+pln.machine         = 'Generic';
+
+pln.numOfFractions  = 30;
+
+% beam geometry settings
+pln.propStf.bixelWidth = 5;
+
+% optimization settings
+pln.propOpt.bioOptimization = 'none';
+pln.propOpt.runVMAT         = true;
+pln.propOpt.runDAO          = true;
+pln.propOpt.runSequencing   = true;
+pln.propOpt.preconditioner  = true;
+pln.propOpt.numLevels       = 7;
+ 
+pln.propOpt.VMAToptions.machineConstraintFile = [pln.radiationMode '_' pln.machine];
+
+pln.propOpt.VMAToptions.maxGantryAngleSpacing    = 2;      % Max gantry angle spacing for dose calculation
+pln.propOpt.VMAToptions.maxDAOGantryAngleSpacing = 4;      % Max gantry angle spacing for DAO
+pln.propOpt.VMAToptions.maxFMOGantryAngleSpacing = 28;     % Max gantry angle spacing for FMO
```

Comment:
These parameters only seem to work for certain combinations of parameters.
Using the following comination throws an error:
``pln.propOpt.VMAToptions.maxGantryAngleSpacing    = 8;      % Max gantry angle spacing for dose calculation
pln.propOpt.VMAToptions.maxDAOGantryAngleSpacing = 16;      % Max gantry angle spacing for DAO
pln.propOpt.VMAToptions.maxFMOGantryAngleSpacing = 28;     % Max gantry angle spacing for FMO``
If this is intended behavior, the conditions should be tested for and a sensible error message should appear. I don't know if I induced this or it was already there before.